charts/prometheus-customizations/templates/prometheus_config.yml

```diff
@@ -0,0 +1,290 @@
+---
+apiVersion: secrets.mz.com/v1alpha1
+kind: ConfigMapSecret
+metadata:
+  name: prometheus-config
+  labels:
+    app: prometheus
+spec:
+  vars:
+    {{- if .Values.influxdb.enabled }}
+    - name: INFLUX_URL
+      secretValue:
+        name: influx-secret
+        key: influxurl
+    {{- else }}
+    []
```

Comment:
does this leave it open for helm instatiations to define more secrets in this via values.yaml, or is the expectation in this chart that the only secret is influx_url or none?

Reply:
It's just influx_url for now, Figured if we have to release this helm chart with a new secret, it wouldn't be the worst thing in the world.

Reply:
makes sense. thanks!